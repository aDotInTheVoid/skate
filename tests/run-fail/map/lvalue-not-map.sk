fn main() {
    // Check error is about z, not x not being map
    let x = 0;
    x.y = z;
}