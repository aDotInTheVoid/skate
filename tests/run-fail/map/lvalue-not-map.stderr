error: Couldn't find variable `z` in scope
  ┌─ $DD/tests/run-fail/map/lvalue-not-map.sk:4:11
  │
4 │     x.y = z;
  │           ^
  │
  = Variables in scope: ["x"]

